README.md

```diff
@@ -61,7 +61,8 @@ In Linux the fuse layer can also be started, stopped and restarted using shell s
 Optional arguments:
 * m : mount point path, default value: /tmp/mnt `Note: Ensure that the mount point path exists`
 * cache : the maximum size of the cache, default value: 100 `Means 100 * 10 MB = 1000 MB`
-* c : connections, download using specified number of connections, default value: 4
+* c : total max connections, total number of API connections used by the application, default value: 4
+* cpf : connections per file, download a file using only specified number of connections, default value: 2, `preferably, c (total max connections)  >= cpf (connections per file)`
```

Comment:
I suggest:
`cpf : connections per file, download a file using the specified number of connections, default value: 2`